### PR TITLE
Enabling JDWP tests

### PR DESCRIPTION
**Pull request by sobirada**
This PR enables JDWP tests on all the platforms for IBM Jdk8, openjdk8, openjdk11 & openjdk17 except on Aixppc platform.

Fixes: runtimes/backlog/issues/477

Signed off by : Soumya Biradar [sobirada@in.ibm.com](mailto:sobirada@in.ibm.com)

### CODE REVIEW

**Review by karianna — APPROVED** [review]
LGTM

**Comment by Mesbah-Alam** [comment]
Re : https://github.com/adoptium/aqa-tests/pull/4106#pullrequestreview-1168645325 - second that. Please do not merge this PR until all intermittent issues are resolved (re: backlog/477).

File: jck/jtrunner/JavaTestRunner.java [hunk 1]
```diff
@@ -671,8 +671,20 @@ public static boolean jckConfigurationForMultijvmWithNoAgent() throws Exception
 						!tests.contains("api/org_omg/PortableServer") ) {
 					fileContent += "set jck.env.runtime.remoteAgent.passiveHost localhost" + ";\n";
 					fileContent += "set jck.env.runtime.remoteAgent.passivePortDefault Yes" + ";\n";
-				}
-			}
+					}
+
+					}
+						
+			// if ( tests.contains("vm/jdwp") )
```
Comment (karianna) [inline jck/jtrunner/JavaTestRunner.java:678]: commented out code?

**Review by karianna — CHANGES_REQUESTED** [review]

**Comment by sobirada** [comment]
@llxia @karianna I still see few issues with grinder submission, working on it.

**Comment by llxia** [comment]
Since it is WIP, I will covert this PR to draft.

**Comment by karianna** [comment]
@sobirada Looks like you need to sign the ECA as well.

**Comment by smlambert** [comment]
Suggest we close this as stale in favour of a fresh PR.  Please reopen as you see fit (with changes and ECA ready).

**Pull request closed without merging**